unit_test/sparse/Test_Sparse_block_gauss_seidel.hpp

```diff
@@ -212,7 +213,7 @@ void test_block_gauss_seidel_rank1(lno_t numRows, size_type nnz,
   // this makes consecutive 5 rows to have same columns.
   // it will add scalar 0's for those entries that does not exists.
   // the result is still a point crs matrix.
-  KokkosKernels::Impl::kk_create_blockcrs_formated_point_crsmatrix(
+  KokkosSparse::Impl::kk_create_blockcrs_formated_point_crsmatrix(
```

Comment:
Maybe try using your IDE's refactor feature to easily fix the `formated` typo Brian found.